### PR TITLE
re add phone number a/b test

### PR DESCRIPTION
**Pull request by calellowitz**
@NoahCarnahan @biyeun cc: @sravfeyn 
This just reverts the pr that took out the phone number a/b test since sales wants it back in now.

### CODE REVIEW

File: corehq/apps/analytics/ab_tests.py [hunk 1]
```diff
@@ -50,3 +50,13 @@ def context(self):
...
     (LIVE_PREVIEW_ENABLED, LIVE_PREVIEW_DISABLED)
 )
+
+NEW_USER_NUMBER_OPTION_SHOW_NUM = 'show_number'
+NEW_USER_NUMBER_OPTION_HIDE_NUM = 'hide_number'
+
+
+NEW_USER_NUMBER = ABTestConfig(
+    'New User Phone Number',
+    'new_phone_aug2016',
```
Comment (NoahCarnahan) [inline corehq/apps/analytics/ab_tests.py]: I wonder if this id should be different? Not sure if product/sales wants this to appear as the same test or a new one...

**Review by NoahCarnahan — APPROVED** [review]
Looks good to me, just one comment (that might not require any action)

**Comment by sravfeyn** [comment]
LGTM pending noah's comment

**Comment by calellowitz** [comment]
@NoahCarnahan I think that slug is only used in the cookie (so shouldnt matter), but just in case i updated it

**Pull request merged**